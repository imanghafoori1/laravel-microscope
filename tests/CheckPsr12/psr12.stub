<?php

namespace CheckPsr12;

class psr12
{
    function hello(){}

    static function hello2() {}

    static public function hello3() {}
}