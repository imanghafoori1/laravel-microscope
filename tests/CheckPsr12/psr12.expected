<?php

namespace CheckPsr12;

class psr12
{
    public function hello(){}

    public static function hello2() {
        }

    static public function hello3() {}
}